lab_1_classify_by_unigrams/main.py

```diff
@@ -63,6 +122,25 @@ def detect_language(
     :param profile_2: a dictionary of a known profile
     :return: a language
     """
+    if not (
+            isinstance(unknown_profile, dict)
+            and isinstance(profile_1, dict)
+            and isinstance(profile_2, dict)
+    ):
+        return None
+
+    profile_1_metric = compare_profiles(unknown_profile, profile_1)
+    profile_2_metric = compare_profiles(unknown_profile, profile_2)
+
+    if (isinstance(profile_1_metric, float)
+            and isinstance(profile_2_metric, float)
+    ):
+        if profile_1_metric > profile_2_metric:
+            return str(profile_2['name'])
+        if profile_1_metric < profile_2_metric:
+            return str(profile_1['name'])
+
+    return [profile_1['name'], profile_2['name']].sort()
```

Comment:
Is this returned when `compare_profile` returned `None`? If so, this is incorrect.

Reply:
if i change something in `detect_language` i do not pass all tests.... i do not know how i can change it:(

Reply:
also all tests are completed, i thought it was okey

Reply:
The logic is as in the description: if obtained values are incorrect, `None` is returned. Judging by what I see, your function won't return `None` in such case

Reply:
fixed